src/frontend/web_application/src/components/ContactProfile/components/ContactProfileForm/index.jsx

```diff
@@ -54,13 +54,13 @@ class ContactProfileForm extends Component {
     this.setState({ contact: { ...this.state.contact, [event.target.name]: event.target.value } });
   }
 
-  handleInfosChanges = (event) => {
+  handleBirthdayChanges = (date) => {
     this.setState({
```

Comment:
setting a new React Component state using the previous state must be done using the updater callback syntax:
https://facebook.github.io/react/docs/react-component.html#setstate